doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -59,6 +59,16 @@ Disable automatic updates if you want to use a custom image for any component of
 ====
 If you use RH-SSO or LDAP authentication, the same user must be configured in your authentication system with the `kie-server,rest-all,admin` roles for {PRODUCT}.
 ====
+. To set the start up strategy, in the *Startup Strategy* field, enter one of the following:
```

Comment:
can we also add a small description about the difference between controller and openshift startup strategy?